examples/error_estimates_forces.jl

```diff
@@ -0,0 +1,182 @@
+# # Practical error bounds for the forces
+#
+# This is a simple example showing how to compute error estimates for the forces on a
+# ``{\rm TiO}_2`` molecule, from which we can either compute asymptotically valid error
+# bounds or increase the precision on the computation of the forces.
+#
+# The strategy we follow is described with more details in[^CDKL2021] and we
+# will use in comments the density matrices framework. We will also needs
+# operators and functions from
+# [`src/scf/newton.jl`](https://dftk.org/blob/master/src/scf/newton.jl).
+#
+# [^CDKL2021]:
+#     E. Cancès, G. Dusson, G. Kemlin, and A. Levitt
+#     *Practical error bounds for properties in plane-wave electronic structure
+#     calculations* Preprint, 2021. [arXiv](https://arxiv.org/abs/2111.01470)
+using DFTK
+using LinearAlgebra
+using ForwardDiff
+using LinearMaps
+using IterativeSolvers
+
+# ## Setup
+# We setup manually the ``{\rm TiO}_2`` configuration from
+# [Materials Project](https://materialsproject.org/materials/mp-2657/).
+Ti = ElementPsp(:Ti, psp=load_psp("hgh/lda/ti-q4.hgh"))
+O = ElementPsp(:O, psp=load_psp("hgh/lda/o-q6.hgh"))
+atoms = [Ti => [[0.5, 0.5, 0.5],
+                [0.0, 0.0, 0.0]],
+         O  => [[0.19542, 0.80458, 0.5],
+                [0.80458, 0.19542, 0.5],
+                [0.30458, 0.30458, 0.0],
+                [0.69542, 0.69542, 0.0]]]
+lattice = [[8.79341  0.0      0.0];
+           [0.0      8.79341  0.0];
+           [0.0      0.0      5.61098]];
+
+# We apply a small displacement to one of the ``\rm Ti`` atoms to get nonzero forces.
+el, pos = atoms[1]
+atoms[1] = Pair(el, pos .+ [[-0.22, 0.28, 0.35] / 10, [0, 0, 0]]);
+# We build a model with one k-point only, not too high `Ecut_ref` and small tolerance to
+# limit computational time. These parameters can be increased for more precise
+# results.
+model = model_LDA(lattice, atoms)
+kgrid = [1, 1, 1]
+Ecut_ref = 35
```

Comment:
Is there any chance you can make that a bit smaller still? Just because it adds about 3 minutes to our CI runtime. It is a nice showcase, so I'd keep it as is if needed.

Reply:
I think we need to figure this out more generally. As we go towards more complicated systems / postprocessings, we can't really expect all examples to run in reasonable time on CI. How about we just give up running the examples on CI and just pre-generate them? Of course it means that they won't be run as CI, so we'd have to move tests to the testsuite and not rely on examples (eg some tests can just be a reduced version of the examples).

Reply:
I tried to use smaller `Ecut_ref` but then the results are not really convincing : the reference solution is not converged (it is not clear if it is converged), and with smaller `Ecut` then the linear regime is not established.

Reply:
I see. Ok, let's leave it as is for now and we try to figure out something more clever to do with this issue in general.